src/reducers/index.ts

```diff
@@ -47,11 +47,28 @@ function parseVega(state: State, action: SetVegaExample | UpdateVegaSpec | SetGi
       vegaSpec: spec,
     };
   } catch (e) {
+    const code = action.spec;
+    const pattern = /\d+/;
```

Comment:
#### Before
```
const pattern = /(end|\d+)/
```
#### Now
```
const pattern = /\d+/
```
I only needed to match numbers. This change required me to force-push the branch.